Source/Core/Core/HW/DSPHLE/UCodes/AXWii.h

```diff
@@ -37,18 +37,13 @@ class AXWiiUCode : public AXUCode
   int m_samples_aux3[6 * 3];
 
   // Are we implementing an old version of AXWii which still has updates?
-  bool m_old_axwii;
+  // bool m_old_axwii;
```

Comment:
I think we want to get rid of this completely instead of just commenting it out, right?

Reply:
Heh,  just noticed that as you commented.